WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartType.kt

```diff
@@ -53,8 +54,8 @@ sealed class QuickStartType(
         return when (this) {
             is NewSiteQuickStartType -> !isQuickStartCompleted &&
                     quickStartStore.hasDoneTask(siteLocalId, QuickStartNewSiteTask.CREATE_SITE)
-            // TODO: ashiagr update isQuickStartInProgress condition for ExistingSiteQuickStartType
-            is ExistingSiteQuickStartType -> !isQuickStartCompleted
+            is ExistingSiteQuickStartType -> quickStartStore.isQuickStartStatusSet(siteLocalId) &&
+                    !isQuickStartCompleted
         }
```

Comment:
That’s because we did not save quick start status (completed = false) earlier when quick start started. For sites with no record for quick start status, `isQuickStartCompleted` will be false as there won't be quick start status present in db which will return incorrect quick start progress status for the existing site. 
Let's keep the existing logic for now. I'll see if I can improve it. 

Thanks for calling it out! 🙇‍♀️